.github/workflows/test.yml

```
@@ -25,6 +25,12 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install MacOS dependencies
+        if: matrix.os == 'macos-latest'
```

I think this needs to be...

```suggestion
        if: ${{ matrix.os }} == 'macos-latest'
```